packages/arb-avm-cpp/avm/include/avm/avmfwd.hpp

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright 2019-2020, Offchain Labs, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
```

Comment:
maybe a better name? not super apparent what the class is supposed to be.

Reply:
Good call. I'm gonna take a look

Reply:
Changed this over to be called `status.hpp`